File: 프로그래머스/unrated/181898. 가까운 1 찾기/가까운 1 찾기.cpp
#include <string>
#include <vector>

using namespace std;

int solution(vector<int> arr, int idx) {
    int answer = 0;
    int max = -1;
    for (int i = 0; i < arr.size(); i++)
    {
        if (arr[i] == 1 && i >= idx)
        {
            max = i;
            break;
        }
    }
    
    return max;
}

File: 프로그래머스/0/181898. 가까운 1 찾기/가까운 1 찾기.java
import java.util.stream.IntStream;

class Solution {
    public int solution(int[] arr, int idx) {
        return IntStream.range(0, arr.length)
                .skip(idx)
                .filter(i -> arr[i] == 1)
                .findFirst()
                .orElse(-1);
    }
}